general/README.md

```diff
@@ -579,3 +580,104 @@ const eventbrite = {
 };
 ```
 **[⬆ back to top](#table-of-contents)**
+
+## Destructuring
+
+The unpacking of values from arrays or properties from objects, into distinct variables. 
+
+In general whenever possible, it is strongly encouraged to use destructuring when accessing 
+values. It allows for easier to maintain code, less if statements, and smaller margin 
+of error. 
+
+### Objects
+
+Always destructure as far as you are able when accessing multiple properties of an object.
+
+*Why do we do this?*: 
+It prevents the need for creating temporary references to those properties, and creates easier to maintain code.
+
+*Exception*: 
+If the object represents empty values with something other than undefined (such as `null`), then use 
+destructuring with caution as defaulting values will not work.
+
+```javascript
+// bad
+const getRicksAsString = (ricks) => {
+    let c137 = ricks.c137;
+    let c186 = ricks.c186;
+
+    return `${c137}, ${c186}`;
+}
+
+// good
+const getRicksAsString = (ricks) => {
+    let {
+        c137, 
+        c186,
+    } = ricks;
+
+    return `${c137}, ${c186}`;
+}
+
+//best (and since no work needs to be done in the body, can implicit return!)
+const getRicksAsString = ({c137, c186}) => (`${c137}, ${c186}`);
+```
+
```

Comment:
Should probably link to the Functions docs that explain about implicit returns work: https://github.com/eventbrite/javascript/blob/master/es6/README.md#arrow-functions